tests/test_flake8_trio.py

```diff
@@ -0,0 +1,66 @@
+import ast
+import os
+import site
+import sys
+import unittest
+from pathlib import Path
+from typing import Any, List, Tuple, Type
+
+from hypothesis import HealthCheck, given, settings
+from hypothesmith import from_grammar
+
+from flake8_trio import TRIO100, Error, Plugin, Visitor
+
+
+class Flake8TrioTestCase(unittest.TestCase):
+    def errors(self, *errors: Error) -> List[Tuple[int, int, str, Type[Any]]]:
+        return [e.values() for e in errors]
+
+    def test_tree(self):
+        plugin = Plugin(tree=ast.parse(""))
+        errors = list(plugin.run())
+        self.assertEqual(errors, [])
+
+    def test_trio100(self):
+        filename = Path(__file__).absolute().parent / "trio100.py"
+        plugin = Plugin(filename=str(filename))
+        errors = list(plugin.run())
```

Comment:
Let's wrap this up in a helper `def get_errors_for_filename(fname: str) -> ast.AST:`, since we'll be doing it a lot.

Or even `def assert_expected_errors(self, fname: str, *expected: Tuple[...]) -> None:` to cover *all* the boilerplate.

Reply:
yeah was definitely gonna do that sooner or later, but I thought I might wait until I have a few more tests implemented to see how much of the work is identical between them and the best way to lay out the helper function. But I can push in an initial version now